GATEWAY/src/providers/tilesApi.service.ts

```diff
@@ -183,21 +183,37 @@ export class TilesApi {
    * @param {string} virtualTileId - The virtual tile
    * @param {string} applicationId - The application the virtual tile is registered to
    */
-  public pairDeviceToVirualTile = (deviceId: string, virtualTileId: string): Promise<Response> => {
+  public pairDeviceToVirualTile = (deviceId: string, virtualTileId: string): Promise<void> => {
```

Comment:
It does not really matter which one it is, as the tests are set up to pass at the moment
So it is up to you 👍